pkg/errctx/context.go

```diff
@@ -94,6 +94,9 @@ func (ctx *Context) AppendWarning(err error) {
 // It also allows using `errors.ErrorGroup`, in this case, it'll handle each error in order, and return the first error
 // it founds.
 func (ctx *Context) HandleError(err error) error {
+	if err == nil {
```

Comment:
This is to make `HandleError` to return quickly and easier to understand. PTAL @YangKeao